azurerm/internal/services/databasemigration/tests/resource_arm_database_migration_service_test.go

```diff
@@ -0,0 +1,233 @@
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDatabaseMigrationServiceExists(data.ResourceName),
+				),
+			},
+			data.RequiresImportErrorStep(testAccAzureRMDatabaseMigrationService_requiresImport),
+		},
+	})
+}
+
+func TestAccAzureRMDatabaseMigrationService_update(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_database_migration_service", "test")
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMDatabaseMigrationServiceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMDatabaseMigrationService_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDatabaseMigrationServiceExists(data.ResourceName),
+				),
+			},
+			data.ImportStep(),
+			{
+				Config: testAccAzureRMDatabaseMigrationService_complete(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDatabaseMigrationServiceExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.name", "test"),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func testCheckAzureRMDatabaseMigrationServiceExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("Data Migration Service not found: %s", resourceName)
+		}
+
+		id, err := parse.DatabaseMigrationServiceID(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		client := acceptance.AzureProvider.Meta().(*clients.Client).DatabaseMigration.ServicesClient
+		ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+		if resp, err := client.Get(ctx, id.ResourceGroup, id.Name); err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: Data Migration Service (Service Name %q / Group Name %q) does not exist", id.Name, id.ResourceGroup)
+			}
+			return fmt.Errorf("Bad: Get on ServicesClient: %+v", err)
+		}
+
+		return nil
+	}
+}
+
+func testCheckAzureRMDatabaseMigrationServiceDestroy(s *terraform.State) error {
+	client := acceptance.AzureProvider.Meta().(*clients.Client).DatabaseMigration.ServicesClient
+	ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_database_migration_service" {
+			continue
+		}
+
+		id, err := parse.DatabaseMigrationServiceID(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		if resp, err := client.Get(ctx, id.ResourceGroup, id.Name); err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: Get on ServicesClient: %+v", err)
+			}
+		}
+
+		return nil
+	}
+
+	return nil
+}
+
+func testAccAzureRMDatabaseMigrationService_base(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-dms-%d"
```

Comment:
Given this is DataBase migrations service, could we maybe use 
```suggestion
  name     = "acctestRG-dbms-%d"
```

WDYT?